bindings/kotlin/example/app/src/main/kotlin/com/lni/example/MainActivity.kt

```
@@ -306,6 +446,34 @@ suspend fun getStrikeBalance(apiKey: String): String = withContext(Dispatchers.I
+    sb.appendLine("=== Generate Mnemonic ===\n")
+
+    try {
+        val wordCount: UByte? = if (use24Words) 24u else null
+        val mnemonic = generateMnemonic(wordCount)
+        
+        val words = mnemonic.split(" ")
+        sb.appendLine("✓ Generated ${words.size}-word mnemonic:\n")
+        
+        // Display words in a numbered list
+        words.forEachIndexed { index, word ->
+            sb.appendLine("${String.format("%2d", index + 1)}. $word")
```

detekt (1.23.8)

[warning] String.format("%2d", index + 1) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)